README.md

```diff
@@ -104,7 +104,7 @@ First, add the `dropwizard-metrics-datadog` dependency in your POM:
 
 ~~~xml    
     <dependency>
-        <groupId>org.coursera</groupId>
+        <groupId>pro.streem</groupId>
```

Comment:
🔧 The group ID should be more specific. Something like `pro.streem.metrics-datadog`, so that the full coordinates look like `pro.streem.metrics-datadog: dropwizard-metrics-datadog:1.1.13`.